assets/js/components/Work/Tabs/Header.jsx

```diff
@@ -1,9 +1,30 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import PropTypes from "prop-types";
 
 const WorkTabsHeader = ({ title, children }) => {
+  const [headerHeight, setHeaderHeight] = useState();
+  const styles = {
+    headerStyle: {
+      position: "-webkit-sticky",
+      position: "sticky",
+      top: headerHeight,
+      zIndex: 1110,
+      padding: "0 1rem",
+    },
+  };
+
+  useEffect(() => {
+    const headerElement = document.getElementById("main-navigation")
+      ? document.getElementById("main-navigation").getBoundingClientRect()
+      : {};
+    setHeaderHeight(headerElement.height);
```

Comment:
No big deal, but you could also write this as:
```
setHeaderHeight(document.getElementById("main-navigation")
      ? document.getElementById("main-navigation").getBoundingClientRect()
      : 0
);
```

As is, it'll set header height to `undefined` if the `headerElement` doesn't exist for some reason.